### ISSUE TITLE
Parsing error with nested if()?

### ISSUE THREAD

**Issue by mpaluchowski** [cflint/CFLint#229]
For this file, named `Test.cfc`:

```cfml
/**
* Some explanation.
*/
component {

  /**
  * Some function explanation.
  *
  * @localData Something.
  * @keyToCheck Something else.
  */
  public string function extractInfo(required struct localData, required string keyToCheck) {
    if (!isNull(localData)) {
      if (structKeyExists(localData, keyToCheck)) {
          return localData[keyToCheck];
      }
    }
  }
}
```

CFLint reports the following:

```shell
$ java -jar CFLint-0.11.1-all.jar -file Test.cfc
ERROR StatusLogger No log4j2 configuration file found. Using default configuration: logging only errors to the console.
Attempt to process expression twice aborted.  This may be a parsing bug.
<issues version="0.11.1">
</issues>
````

The `Attempt to process expression twice aborted.  This may be a parsing bug.` is triggered by both of the `if` lines together, because if I remove either of the, the message disappears. I tried moving around some brackets, spaces etc. to pinpoint the exact cause, but the message staid.

**Comment by ryaneberly** [comment]
@mpaluchowski , thanks for the clear code sample.  It was a false positive.  It is fixed.
